differ/differ.go

```diff
@@ -793,13 +825,22 @@ func startDiffer(config conf.ConfigStruct, storage *DBStorage, verbose bool) {
 	log.Info().Int(clustersAttribute, entries).Msg("Read cluster list: done")
 	log.Info().Msg(separator)
 	log.Info().Msg("Read previously reported issues for cluster list")
-	previouslyReported, err = storage.ReadLastNotifiedRecordForClusterList(clusters, notifConfig.Cooldown)
+	previouslyReportedNotificationBackend, err = storage.ReadLastNotifiedRecordForClusterList(
+		clusters, notifConfig.Cooldown, types.NotificationBackendTarget)
 	if err != nil {
 		ReadReportedErrors.Inc()
 		log.Err(err).Msg(operationFailedMessage)
 		os.Exit(ExitStatusStorageError)
 	}
-	log.Info().Int("previously reported issues still in cooldown", len(previouslyReported)).Msg("Get previously reported issues: done")
+	previouslyReportedServiceLog, err = storage.ReadLastNotifiedRecordForClusterList(
+		clusters, notifConfig.Cooldown, types.ServiceLogTarget)
```

Comment:
cooldown won't be used there

Reply:
+1

Reply:
PTAL https://github.com/RedHatInsights/ccx-notification-service/pull/243/commits/ecd3b4086b4235f50e973c69dcb23c750ec51c94